Day 2 - Python Datetime objects and Timedelta/solution/solution.py

```diff
@@ -0,0 +1,11 @@
+from datetime import datetime, timedelta
+
+christmas_date = datetime(year=2019, month=12, day=25)
+
+# 3 weeks and 4 days time can be interpreted as 3*7days+4days = 25 days
```

Comment:
There is also an argument for `weeks`. If you use that, you don't have to calculate the number of days.